### ISSUE TITLE
Edits of logged in person are not permanent

### ISSUE THREAD

**Issue by PiQuer** [inbaz/ers#421]
To reproduce: log in, change the person, log out, log in, changes are reverted.

**Comment by inbaz** [comment]
Good point, maybe this is fixed with the last database changes. Needs to be rechecked.

**Comment by Tobias-Horstmann** [comment]
I'm not exactly sure what "change person means"? Does it mean personal details of the person?

I tested in version 0.4.52. and logged in, changed personal details (e.g. name), logged out and logged in again. Changes were still there, works fine.
